sway/input/seat.c

```diff
@@ -1462,6 +1462,18 @@ struct sway_node *seat_get_focus(struct sway_seat *seat) {
 	return current->node;
 }
 
+struct wlr_surface *seat_get_focused_surface(struct sway_seat *seat) {
```

Comment:
Ideally this function would also handle the case where a lockscreen has focus.

Reply:
Is a lockscreen not covered by layershell? What do I need to do here?